package/debian/control

```diff
@@ -0,0 +1,15 @@
+Source: hyper
+Section: unknown
+Priority: optional
+Maintainer: Xianglin Gao <xlgao@zju.edu.cn>
+Build-Depends: debhelper (>= 8.0.0)
+Standards-Version: 3.9.4
+Homepage: <insert the upstream URL, if relevant>
+#Vcs-Git: git://git.debian.org/collab-maint/hyper.git
+#Vcs-Browser: http://git.debian.org/?p=collab-maint/hyper.git;a=summary
```

Comment:
not quite sure.

I don't know if there could be a github url. For safety, we can remove the two lines.